mm/mixtures.py

```diff
@@ -100,6 +103,11 @@ def merge_helper(elements):
                 "merge.log.%d" % merge_num, self.merge_log_file[merge_num].get_path()
             )
 
+            for e in elements:
+                if e in tmp_files:
+                    logging.info("unlink %s" % e)
+                    os.unlink(e)
+
```

Comment:
after unlinking the files should you remove them from `tmp_files`?

Reply:
possible, but not necessary. tmp_files is just an allow-list for elements to delete.